src/wordlookup/wiktionaryel.py

```diff
@@ -56,28 +65,43 @@ async def create(cls, word: str):
                 page_content = await response.read()
 
         soup = BeautifulSoup(page_content, "html.parser")
-        WiktionaryQuery.remove_ancient_greek(soup)
+        WiktionaryQuery.remove_ancient_greek(soup, language)
+
         self.soup = soup
 
         return self
 
     @staticmethod
-    def remove_ancient_greek(soup: Any) -> None:
+    def remove_ancient_greek(soup: Any, language: str) -> None:
         # find where the ancient greek section begins
-        tag_to_remove = soup.find("span", id="Αρχαία_ελληνικά_(grc)")
+        if language == "english":
+            remove_string = "Ancient_Greek"
+            stop_at_string = "Greek"
+        else:
+            remove_string = "Αρχαία_ελληνικά_(grc)"
+            stop_at_string = None
+
+        tag_to_remove = soup.find("h2", id=remove_string)
+
         if tag_to_remove:
-            parent = tag_to_remove.find_parent()
-            # remove everything under ancient section (always below modern)
-            for sibling in parent.find_next_siblings():
-                sibling.extract()
+            current_element = tag_to_remove.find_parent()
+            while current_element:
+                next_sibling = current_element.find_next_sibling()
+                if next_sibling and stop_at_string and next_sibling.find("h2", string=stop_at_string):
+                    break
+
+                # remove the current element and move to next sibling
+                current_element.extract()
```

Comment:
Does this line do anything?

Reply:
yeah it removes the ancient greek entries so you don't get a muddle of ancient greek and modern greek entries. the break is to stop the modern greek entries being removed on the en pages as the order is switched.